### PR TITLE
feature: improve translations

### LINKED ISSUE #1018: Improve translations
**Description**
Improve translations structure and apply a key naming convention. Different scopes should be in different files for better loading.

#### Definition of done
- [ ] Description is created (Task creator)
- [ ] Code is complete (Developer)
- [ ] Scope of test and what was done section is defined in task (Developer)
- [ ] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Fixes #1018

This PR divides translations into muitple files, improves translations keys, fixes language detection and adds new messages

Full translations outside EN are yet to be completed. Unused keys should be deleted.

# To Test

1.  Open every page
- [ ] Check if there aren't keys without message
2. Change browser language
- [ ] Check if dApp detects changes and show some broswer language translations

### CODE REVIEW

File: public/locales/de/bridge.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (jorgedclopes) [inline public/locales/de/bridge.json:1]: de?
I thought we were only supporting en!!

**Comment by wixzi** [comment]
👺  Aswm job man. Splitting the translations was a great idea. 👏🏼 . I will give it a try and approve.

File: public/locales/de/bridge.json (hunk 1 above)
Comment (berteotti) [reply public/locales/de/bridge.json:1]: We have other languages that are semi-translated. Untranslated keys outside EN must be in english.

File: public/locales/en/swap.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (Diogomartf) [inline public/locales/en/swap.json:1]: same, shouldn't this keys have the `swap` prefix?

**Comment by wixzi** [comment]
Screenshot 2022-07-08 at 16 56 49
@berteotti What do you think about adding keys to all main links. Now I think only `swap` has german translation

**Comment by wixzi** [comment]
Also we should write a test case to make sure all the translations has all the keys. Can be a later improvement

**Comment by wixzi** [comment]
@berteotti  Awsm [9a9be43](https://github.com/levelkdev/swapr-dapp/pull/1199/commits/9a9be4343e4d6ad13f013cbdbf7e7d59b00ec777) ✨

File: public/locales/en-US/liquidity.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "liquidityMining.header": "Liq. mining",
+  "liquidityMining.title": "Liquidity mining",
+  "liquidityMining.action.create": "CREATE LIQ. MINING",
```
[the remark this answers is not in the document]
Comment (berteotti) [reply public/locales/en-US/liquidity.json]: Fixed! This key wasn't being used.

**Comment by niemam29** [comment]
As the translations are not ready i just checked if the language switching is working and lgtm

File: public/locales/en/bridge.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "bridgeModal.modalText": "{{typedValue}} {{symbol}} from {{fromNetworkName}} to {{toNetworkName}}",
```
[the remark this answers is not in the document]
Comment (jorgedclopes) [reply public/locales/en/bridge.json:2]: whatever we decide, we should create documentation for these decisions...

File: public/locales/de/liquidity.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "liquidityMining.header": "Liq. mining",
+  "liquidityMining.title": "Liquidity mining",
```
Comment (jorgedclopes) [inline public/locales/de/liquidity.json:3]:
```suggestion
  "liquidityMining.header": "Liq. Mining",
  "liquidityMining.title": "Liquidity Mining",
```

File: public/locales/en-US/liquidity.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "liquidityMining.header": "Liq. mining",
+  "liquidityMining.title": "Liquidity mining",
```
Comment (jorgedclopes) [inline public/locales/en-US/liquidity.json:3]:
```suggestion
  "liquidityMining.header": "Liq. Mining",
  "liquidityMining.title": "Liquidity Mining",
```

File: public/locales/en/liquidity.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
+{
+  "liquidityMining.header": "Liq. mining",
+  "liquidityMining.title": "Liquidity mining",
```
Comment (jorgedclopes) [inline public/locales/en/liquidity.json:3]:
```suggestion
  "liquidityMining.header": "Liq. Mining",
  "liquidityMining.title": "Liquidity Mining",
```

**Review by jorgedclopes — APPROVED** [review]
I have a couple of comments and suggestions but I don't think they're worth blocking this PR.
Whatever is in here, I think we can fix forward.

**Review by Diogomartf — APPROVED** [review]

File: public/locales/de/liquidity.json (hunk 5 above)
Comment (berteotti) [reply public/locales/de/liquidity.json]: Thanks Jorge, this and similar comments are fixed in [8d38bc1](https://github.com/levelkdev/swapr-dapp/pull/1199/commits/8d38bc14c8f3e95a847d9b09fdb86d1d52c24317)

**Pull request merged**